wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -124,6 +124,7 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
 
     fun init() {
         GlobalScope.launch {
+            delay(5000)
```

Comment:
Hotfix for error `PlatformRepo: lateinit property blockchainIdentity has not been initialized`

Reply:
Line causing the issue: https://github.com/dashevo/dash-wallet/blob/4448baa1a9d8eb49e71ce6bccc878b0ef72ba18c/wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt#L628

Reply:
This issue comes from `evonet-develop` branch

Reply:
Thanks for this report.  It will be fixed in a different PR.